loggins.sh

```diff
@@ -87,7 +91,28 @@ function explain ()
 	fi
 }	
 
+# The ZSHEXIT functions
+function shutdown()
+{
+	echo "Shutting down..."
+	derby stop &> /dev/null
+	set -m
+}
+
+# OTHER USEFULE FUNCTIONS
+function quit()
+{
+	echo "Shutting down..."
+	derby stop &> /dev/null
+	set -m
+	add-zsh-hook -D precmd status
+	add-zsh-hook -D preexec loggins
+	add-zsh-hook -D zshexit shutdown
```

Comment:
Wouldn't disabling logging be better done by starting a new session with it disabled(ie removing from zshrc)?